server/api.js

```diff
@@ -71,6 +72,10 @@ const auth = async (req, res, next) => {
 			return next();
 		}
 
+		if (allowedAccess === user.token) {
+			return next();
+		}
+
```

Comment:
Why is this here? Didn't you already add this in your previous commit?

Reply:
I have removed it